ddl/ddl.go

```diff
@@ -318,7 +318,6 @@ func (d *ddl) Start(ctxPool *pools.ResourcePool) error {
 	logutil.BgLogger().Info("[ddl] start DDL", zap.String("ID", d.uuid), zap.Bool("runWorker", RunWorker))
 	d.ctx, d.cancel = context.WithCancel(d.ctx)
 
-	d.wg.Add(1)
```

Comment:
moved into `limitDDLJobs`

Reply:
Why put it into `limitDDLJobs `? If put it into `limitDDLJobs`,  how about the case of calling `d.wg.Wait()` before `go d.limitDDLJobs()`.

Reply:
It looks clear to move `d.wg.Add(1)` and `d.wg.Done()` together since currently all call to `d.wg.Add(1)` are following `go d.limitDDLJobs()`.

I'm not sure what's the required sequence of "calling `d.wg.Wait()` before `go d.limitDDLJobs()`". If we need manually 

- `wg.Add`
- do something
- `wg.Wait`
- run `limitDDLJobs`

then what are we waiting for? Currently the only `wg.Done` is in `limitDDLJobs`, so that means we are waiting for previous `limitDDLJobs` finishing. That should equal to put `wg.Add` into `limitDDLJobs` and doesn't need us manually call it.